client/src/pages/DuelOverview.js

```diff
@@ -110,14 +110,16 @@ class DuelOverview extends Component {
           <UserBadge user={opponent} />
         </header>
 
-        <Link
-          to="#" // TODO
-          className="btn"
-          onClick={() => alert("Oups, tu as trouvé une fonctionnalité pas encore implémentée !")}
-          disabled={!currentUserCanPlay}
-        >
-          Jouer le tour {rounds.length}
-        </Link>
+        {currentUserCanPlay &&
+          <Link
+            to={`/duel/${this.props.match.params.id}/play`}
+            className="btn"
+            disabled={!currentUserCanPlay}
+          >
+            Jouer le tour {rounds.length}
+          </Link>
+        }
```

Comment:
I wanted to let the button shown, but disabled, to keep UI coherence. Is that a good idea?

Reply:
No, the button is useless

Reply:
Yes but the player can wonder "where is the play button", thinking that it's elsewhere. By putting a disabled button, we make clear "the button is here but you can't play ATM"

Reply:
That's not the same view for the user. There is an intro view, and a summary view

Reply:
I don't understand why you say it's not the same view. It have the same overall look (score and avatars on the top, bricks with each rounds). Plus it's accessible on `/duel/:id`

Reply:
You don't access it at the same moment
Intro -> when you will begin a round
Summary -> A pending challenge or the summary of a round

And the user won't access to the page by the url, so that's not important

Reply:
We are talking about the duel overview. We access it when we want to see an overview of the duel, with the passed and current rounds (if applicable)
It's the same page (= the same URL), and if a user refresh the page, they can see the button appearing (if the other player play the round between to refreshes).

We may have a misunderstood of the voc here. To me : 
- a duel = multiple rounds
- a round = 5 questions per player
- a challenge = ??

Reply:
Okay but i don't need that button
grrrrrr